spec/features/admin/customers_spec.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
-require 'spec_helper'
+require 'system_helper'
```

Comment:
I would have done this in the same commit as moving the file. Otherwise you have a commit which puts the code in an invalid state. File in spec/features should not use the system helper. :wink: